src/Algebra/Graph/Relation/InternalDerived.hs

```diff
@@ -66,27 +59,146 @@ The 'Show' instance produces symmetrically closed expressions:
 
 @show (1     :: SymmetricRelation Int) == "vertex 1"
 show (1 * 2 :: SymmetricRelation Int) == "edges [(1,2),(2,1)]"@
```

Comment:
I think the code above (which was written by me) is incorrect too: we should show `1 * 2` as `edge 1 2`.

Reply:
should I just change to `edge 1 2`?

Reply:
Yes.